runtime/doc/autocmd.txt

```diff
@@ -758,6 +758,15 @@ LspNotify			See |LspNotify|
 LspProgress			See |LspProgress|
 LspRequest			See |LspRequest|
 LspTokenUpdate			See |LspTokenUpdate|
+							*MarkSet*
+MarkSet				Just after a |mark| is set. The |event-data|
```

Comment:
What is set as the "pattern"? Presumably plugins may want to listen to specific marks only (for performance):

```lua
vim.api.nvim_create_autocmd({'MarkSet'}, {
  pattern = {'[', ']'},
  command = "echo 'foo'",
})
```

Reply:
Currently, the pattern is the filename like other autocommands. I'm a little wary about changing that – are there any other autocmmands that have non-file patterns? – but I do think specifying the marks like this is pretty nice.

Reply:
Updated the pattern to use marks instead of filenames!